instance/models/instance.py

```diff
@@ -455,10 +455,8 @@ class AnsibleInstanceMixin(models.Model):
     """
     An instance that relies on Ansible to deploy its services
     """
-    ansible_source_repo_url = models.URLField(max_length=256,
-                                              default='https://github.com/edx/configuration.git')
-
-    configuration_version = models.CharField(max_length=50, default='master')
+    ansible_source_repo_url = models.URLField(max_length=256, blank=True)
+    configuration_version = models.CharField(max_length=50, blank=True)
```

Comment:
Actually, looking at the `watch_pr` method, this is already how it works, except that it doesn't set the defaults in a pre-save hook yet. And I am not sure what happens if you click the 'reprovision' button on an instance that was created from a PR - is the PR body re-read when reprovisioning? I believe it should be if it currently isn't.